crepecake/src/main/java/com/google/cloud/tools/crepecake/registry/json/ErrorEntryTemplate.java

```diff
@@ -19,6 +19,7 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.google.cloud.tools.crepecake.json.JsonTemplate;
 
+// TODO: Should include detail field as well - need to have custom parser
```

Comment:
issue or fix

Reply:
Filed https://github.com/GoogleCloudPlatform/minikube-build-tools-for-java/issues/113